cPP_DSC.adoc

```diff
@@ -610,19 +610,19 @@ a|[cols=".^1,3",options=noheader,grid="none",frame="none"]
 
 !FCS_CKM.1 !Cryptographic Key Generation
 
-!FCS_CKM.2 !Cryptographic Key Establishment
+!FCS_CKM_EXT.7 !Cryptographic Key Agreement
 
 !FCS_COP.1/Hash !Cryptographic Operation (Hashing)
 
-!FCS_COP.1/HMAC !Cryptographic Operation (Keyed Hash)
+!FCS_COP.1/KeyedHash !Cryptographic Operation (Keyed Hash)
 
-!FCS_COP.1/KAT !Cryptographic Operation (Key Agreement/Transport)
+!FCS_COP.1/KeyEncap !Cryptographic Operation - Key Encapsulation
 
-!FCS_COP.1/KeyEnc !Cryptographic Operation (Key Encryption)
+!FCS_COP.1/KeyWrap !Cryptographic Operation - Key Wrap
```

Comment:
crypto WG has FCS_CKM_EXT.7 which includes the key agreement algorithms. I see all the algorithms listed in the current FCS_COP.1/KAT covered in the combination of FCS_COP.1/KeyEncap and FCS_CKM_EXT.7. Nothing is missing, it has just been split into two based on what they are used for. This likely means we need to adjust how we write the requirements we are expecting some, but there is nothing missing (I couldn't tell you why some of these are as FCS_CKM instead of FCS_COP, but that logic has never made much sense to me).